File: Monotonic Array/O(n)_time_and_O(1)_space.py
def isMonotonic(array):
    # Write your code here.
    nonIncreasing=True
    nonDecreasing=True
    for  i in range(len(array)-1):
        if array[i] < array[i+1]:
            nonIncreasing=False
        if array[i] > array[i+1]:
            nonDecreasing=False
        
    return nonDecreasing or nonIncreasing

File: Strings/Palindrome check/O(n)_time_and_O(1)_space.js
function isPalindrome(string) {
  // Write your code here.
  let start = 0;
  let end = string.length - 1;
  while (start < end) {
    if (string[start] !== string[end]) return false;
    start++;
    end--;
  }
  return true;
}
